ui/components/multichain/account-list-item-menu/account-list-item-menu.js

```diff
@@ -121,6 +125,20 @@ export const AccountListItemMenu = ({
     };
   }, [handleClickOutside]);
 
+  const handlePinning = (address) => {
+    const updatedPinnedAccountList = [...pinnedAccountList, address];
+    dispatch(updateAccountsList(updatedPinnedAccountList));
+  };
+
+  const handleUnpinning = (address) => {
+    const indexToRemove = pinnedAccountList.indexOf(address);
+    if (indexToRemove !== -1) {
```

Comment:
Nit: Can these two functions be one called `togglePinning`? If the index is `-1`, that would be a `pin` action, right?

Reply:
Yup, updated here https://github.com/MetaMask/metamask-extension/pull/21865/commits/8673c6a9d9cdeccc5ca2a5b45b32a171b9dff4cd